dashboards/health.erb

```diff
@@ -508,6 +508,22 @@ $(function() {
         <div data-id="calculate-release-dates-dev" data-view="ServerStatusSquares" data-title="Calculate Release Dates - Dev"></div>
       </li>
 
+      <li data-row="19" data-col="5" data-sizex="1" data-sizey="1">
```

Comment:
This these would be clearer shown in sets of 3? If you check the other services before yours they are shown in threes (though CVL is a group of 2 as no prod env yet).